.vscode/settings.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I appreciate this but prefer not to include IDE config in the codebase. Can you remove?